### ISSUE TITLE
add novnc(html 5) integrated to proxmox module

### ISSUE THREAD

**Issue by IBogdizZ** [blesta/module-proxmox#43]

**Comment by habibulilalbaab** [comment]
I've added it, in pull request #44 
![image](https://github.com/blesta/module-proxmox/assets/28334523/074d1f9d-b40a-40f9-bf14-592a296b9e4a)

**Comment by habibulilalbaab** [comment]
#44

**Comment by IBogdizZ** [comment]
Wow nice

**Comment by IBogdizZ** [comment]
Tried installing it IT was the same maybe i am doing Something wrong

**Comment by IBogdizZ** [comment]
> I've added it, in pull request #44 ![image](https://private-user-images.githubusercontent.com/28334523/295816056-074d1f9d-b40a-40f9-bf14-592a296b9e4a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ5OTQ3OTEsIm5iZiI6MTcwNDk5NDQ5MSwicGF0aCI6Ii8yODMzNDUyMy8yOTU4MTYwNTYtMDc0ZDFmOWQtYjQwYS00MGY5LWJmMTQtNTkyYTI5NmI5ZTRhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTExVDE3MzQ1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFmMWZjNTNhMDE1OGJjOGY5NzVkYjVkMGI5ZWQ0MGRlZjE2NGYyYzQ4ZThjYmRiZjVlYjIzYThhNDgyNDRjMzUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.iyvJieyFwBtGUW7OYlzLZMH__7kjGGhzywTuA5cFjH0)

**Comment by habibulilalbaab** [comment]
yup, need add cookie to make novnc works. in my dev working well now.  I'll update it to repo

**Comment by habibulilalbaab** [comment]
#46 let me know if it works

**Comment by IBogdizZ** [comment]
> yup, need add cookie to make novnc works. in my dev working well now. I'll update it to repo

so i did try to update and it get me a white screen
![image](https://github.com/blesta/module-proxmox/assets/128488448/d053b2d0-9a7c-43ef-ba78-b33f1400483a)

**Comment by habibulilalbaab** [comment]
what are the results from the console? You can replace all files with my repository.
backup the previous module.

I use the latest Blesta, everything works well.

**Comment by IBogdizZ** [comment]
i managed to fixed it
but i get this
![image](https://github.com/blesta/module-proxmox/assets/128488448/6374f54e-2e80-47ec-800a-1f2f5c8e4336)

**Comment by IBogdizZ** [comment]
in console it says refused connection and then my url for the proxmox

**Comment by habibulilalbaab** [comment]
please inspect element & check iframe url
is url correct and accessible to pve server?

**Comment by IBogdizZ** [comment]
yes the url is corect
![image](https://github.com/blesta/module-proxmox/assets/128488448/d3cc6b4e-fafa-43a4-b881-c7d02533f313)
![image](https://github.com/blesta/module-proxmox/assets/128488448/263d7191-323d-4671-8ac7-ef3c6ab5ecc5)


> please inspect element & check iframe url is url correct and accessible to pve server?

**Comment by habibulilalbaab** [comment]
I can't do any further troubleshooting, maybe you can open a new tab on the iframe URL and see what error appears

**Comment by IBogdizZ** [comment]
ok.
